### ISSUE TITLE
AWACS spawning should depend on base ownership at time of spawn

### ISSUE THREAD

**Issue by Winston2108** [ModernColdWar/RedStormRising#101]
Pretty minor, but in theory every half hour when we spawn the AWACS we should check who owns the associated base.  Ie if red captured Vaziani half-way through the 8 hour mission they should start getting the AWACS there and blue stop getting it.

Pretty minor though

**Comment by Winston2108** [comment]
FYI @chandawg; this here as a reminder of something we could improve, but probably won't.  I like to put these things in so we have a record of the stuff we end up not doing as well (if this one ends up in that bucket).

**Comment by chandawg** [comment]
They were spawning based on triggers in the mission?

**Comment by Winston2108** [comment]
No, what I mean is that the check for whether it's a blue or red AWACS at Vaziani or Kras-C should really happen *every time* a fresh AWACS is spawned, rather than just at the start of the mission.

Hypothetically, if Vaziani was red at mission start, and blue capped it 2 hours in, they'd then have their base defences shooting down an A-50 every half an hour thereafter.  In theory, they should get a blue AWACS after the red one is shot down.

This is pretty unimportant, but it's here just so we actively decide to ignore it for now. :-)

**Comment by chandawg** [comment]
Going to look at this, also contemplating having an associated AI escort or intercept?
